docs/reference/api.rst

```diff
@@ -318,9 +318,6 @@ Data
     :undoc-members:
     :show-inheritance:
 
-.. autoclass:: inmanta.data.model.BaseModel
-    :show-inheritance:
-
```

Comment:
This is not `BaseModel.Config`. It seems you already removed `BaseModel.Config` in the past: https://github.com/inmanta/inmanta-core/commit/a13f49965e0f3e26e7ab7c1c4684b1f6a01f1b6d